tests/double_plus_back_error.oomph

```diff
@@ -0,0 +1,2 @@
+export func main():
+    print(thing++)
```

Comment:
These tests are missing `let thing = 1`. Also, the names of the tests should use prefix/suffix instead of front/back